### ISSUE TITLE
Is this project officially abandoned?

### ISSUE THREAD

**Issue by BanCrash** [Kapiainen/Lauhdutin#164]
Hello, it's been more than 2 years since the last commit, so I guess this is an unnecessary question, but is this project abandoned?

Regards.

**Comment by Jadiction** [comment]
It does appear to be so. ;(
Shame because this is the best rainmeter game launcher skin I could find.

Does anyone have any alternatives?
That provides basically this skin - simple scroll list of games with customizable banners & game importer.

**Comment by Nathipha** [comment]
@Jadiction Unfortunately, not really, no. I'm also looking for a small and simple, vertical "skin" that you can keep on one side of your desktop to quickly access games without going through whatever launcher is used (steam, epic, uplay, origin,...). Unfortunately Lauhdutin keeps bugging out more and more (won't load the list when Windows starts up, missing games,...), so I might have to try one of the others in the future. :(

There are other "skins" that add a (horizontal) scroll list but none of them is as simple as Lauhdutin:
1. Gamepad 2.0 ([link](https://www.deviantart.com/starl0rd84/art/Gamepad-2-0-For-Rainmeter-496008344)) - released in 2014
2. GameHUB ([link](https://www.deviantart.com/not-finch/art/GameHUB-launcher-1-5-for-Rainmeter-768260883)) / GameHub launcher 2 ([link](https://www.deviantart.com/not-finch/art/GameHUB-launcher-2-for-Rainmeter-785369648)) - they both add a full screen scrollable list (kind of like the Win 8 start menu)
3. NXT-OS ([link](https://nxtos.com/)) - full skin with a "game drawer" (horizontal list)

**Comment by Jadiction** [comment]
@Nathipha Yeah, those are the same ones I found, along with [GameView 3](https://www.deviantart.com/keystothecity/art/Gameview-3-538954087), but they just aren't what I'm looking for like Lauhdutin provides.

GameVIew was closer to what I was looking for and it had a simple way to add games (just a directory with shortcuts), I'm just not a fan of a list of text. I was going to try to modify it to add banners, but haven't gotten around to it.

**Comment by BanCrash** [comment]
@Nathipha @Jadiction Well, I can't make any promises since everything I know about programming is what I learnt in my free time, but since you are out of options I guess I'm better than nothing xD. Let me know what issues are you having and let's see if I can solve them, or at least some of them.

**Comment by Nathipha** [comment]
@Jadiction Aha, didn't even find GameView3 but you're right, only text isn't what I'm looking for either. I wouldn't mind adding games by hand but it's still nice when it auto-detects everything, including the banner. The problem with GameHub is that it's pretty resource-intensive, at least according to the description, so it's highly unlikely that there's ever going to be a desktop-only version like Lauhdutin.

@BanCrash 
Have you been running Lauhdutin too? It's basically the same thing every time: After Windows boots, usually the game list won't load again. About 75% of the time there's some error message like "failed to read"/"no writing permission"/... and 25% of the time it only loads either the games I manually added or non-Steam games that I added to Steam.
In addition there's currently no support for any of the other big launchers like epic, uplay or origin and even if I add one of their games to Steam, I'm not sure how to add a banner for it.
Of course I'd be thankful if someone would fix these problems but, let's be honest, fixing someone else's code isn't fun, so I wouldn't blame you if you didn't end up doing it.
